source/_integrations/plugwise.markdown

```diff
@@ -43,32 +43,43 @@ Platforms available - depending on your Smile and setup include:
  - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
  - `number` (for changing a boiler setpoint, Circlesa temperature offset)
 
-The password can be found on the bottom of your Smile or Stretch, the ID, it should consist of 8 characters. To find your IP address use the Plugwise App: 
+## Before configuring
 
- - Open the Plugwise App and choose the 'Settings'-icon (&#9776;) and choose 'HTML-interface'. 
- - Go to the (lower) 'Settings'-icon (&#9776;) and choose 'Preferences'. 
- - Choose 'System' then 'Networking' and your IP address will be shown.
+{% important %}
+When you have an Anna and an Adam, make sure to **only** configure the Adam integration. You can press the "IGNORE" button on the auto-discovered Anna integration to hide this integration. In case you need to rediscover the Anna integration, make sure to click the "STOP IGNORING" button on the Plugwise integration first, available via "show ignored integrations".
```

Comment:
I thought this was solved in the integration? As in, it shouldn't auto discover twice?

Reply:
Good call, I'll double-check

Reply:
https://github.com/home-assistant/core/blob/4eb5734d73ee601bbd6223941ea12e8ffe0b3583/homeassistant/components/plugwise/config_flow.py#L124

It still applies though aside from discovery, I'll reword it